sagemaker-pipelines/tabular/tuning-step/tuning_step_example.ipynb

```diff
@@ -0,0 +1,826 @@
+{
```

Comment:
<p>Be consistent with capitalization - should be <code>Hyperparameter</code>  instead of <code>HyperParameter</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>